packages/@okta/vuepress-site/docs/reference/api/device-assuarance-policies/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
File name is incorrect. This is breaking the ToC link.

device-assuarance-policies 🚫 
device-assurance-policies 🆗

Reply:
Even better catch. Merci!